### ISSUE TITLE
Possible to ignore the Axios

### ISSUE THREAD

**Issue by coderabsolute** [ratiw/vuetable-2#201]
Possible not to use Axios but send my own array of an objects and populate the vueTable-2?

**Comment by ratiw** [comment]
@coderabsolute Yes, by using Data mode. However, I didn't have time to finish the tutorial on Data mode, you'll just have to look at [this sample](https://codepen.io/ratiw/project/editor/DzmJnA/).
